security-demo/release/app-binaries/config/app.conf

```diff
@@ -0,0 +1 @@
+placementURL = "ws://127.0.0.1:20000"
```

Comment:
Placement is not required in kubeedge, right?

Reply:
The demo is based on v0.1. So placement mock is required. In the next version, when cloud components are integrated, all config and changes for cloud stub will be removed.